pkgs/applications/audio/pyradio/default.nix

```diff
@@ -11,6 +11,15 @@ python3Packages.buildPythonApplication rec {
     sha256 = "04asw5alkkf2q5iixswarj6ddb0y4a6ixm7cckl6204jiyxpv6kc";
   };
 
+  propagatedBuildInputs = with python3Packages; [
+    dnspython
+    requests
+  ];
+
+  prePatch = ''
```

Comment:
Built it with `nixpkgs-review pr 151584`. Works great. Thanks. Two small thing. Please add a separate commit to add yourself as a maintainer. Also, it would be better if you can format the code with. https://github.com/nix-community/nixpkgs-fmt

Reply:
Alright, hopefully the formatting works now.

How do I add myself as maintainer? Do I create a commit and add myself to the https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix? And then do I create another commit to add myself as a maintainer to pyradio?

Reply:
Yeah.

Reply:
This has way more overhead than imagined, but maybe I got it right this time.

Reply:
Look good to me. It may take a while for some committer to merge it.